TuneUpTests/TuneUpTests.csproj

```diff
@@ -51,19 +51,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="System.Xaml" />
-    <Reference Include="SystemTestServices, Version=2.5.0.6450, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\Dynamo\bin\AnyCPU\Debug\SystemTestServices.dll</HintPath>
-    </Reference>
-    <Reference Include="TestServices, Version=2.5.0.6450, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\Dynamo\bin\AnyCPU\Debug\TestServices.dll</HintPath>
-    </Reference>
     <Reference Include="WindowsBase" />
-    <Reference Include="WorkspaceDependencyViewExtension, Version=2.5.0.6246, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\Dynamo\bin\AnyCPU\Debug\WorkspaceDependencyViewExtension.dll</HintPath>
```

Comment:
Remove these local dependencies for now

Reply:
Will we need them in future?

Reply:
I dont think we TuneUp currently is depending on workspace References extension, the other two test binaries are included in our DynamoVisualProgramming.Test.Nuget